src/bindings/python/src/openvino/frontend/pytorch/patch_model.py

```diff
@@ -79,3 +80,19 @@ def unpatch_model(model, orig_forward_name):
                 print('[ WARNING ] Exception raised during model unpatching. Depending on the exact issue it may lead to broken original model.')
                 print('Original exception details:')
                 print(error)
+
+
+def __make_16bit_traceable(model: torch.nn.Module):
```

Comment:
Should user patch a model manually? No automatic detection mechanism?
Also, will it be internal or external function? If external, do we need `__` in the method name?

Reply:
We decided to make it internal for now